config/environments/development.rb

```diff
@@ -49,4 +49,22 @@
 
   # Raises error for missing translations
   # config.action_view.raise_on_missing_translations = true
+  # logstasher
+  # Enable the logstasher logs for the current environment
+  config.logstasher.enabled = true
+
+  # This line is optional, it allows you to set a custom value for the @source field of the log event
+  config.logstasher.source = 'Advocate Defence Payments App development'
+
+  # This line is optional if you do not want to suppress app logs in your <environment>.log
+  config.logstasher.suppress_app_log = true
+
+  # This line is optional if you do not want to log the backtrace of exceptions
+  config.logstasher.backtrace = true
+
+  # Enable logging of controller params
+  config.logstasher.log_controller_parameters = false
```

Comment:
Did you chose to make this false rather than scrub the parameters we are worried about?